democracy/models/base.py

```diff
@@ -115,6 +115,13 @@ class Meta:
         abstract = True
 
 
+class Organization(StringIdBaseModel):
+    name = models.CharField(verbose_name=_('name'), max_length=255, unique=True)
+    admin_users = models.ManyToManyField(
+        settings.AUTH_USER_MODEL, blank=True, related_name='admin_organizations'
+    )
```

Comment:
IMO this model shouldn't be in `base.py` as it isn't used as a basis for anything